fix: intro (#191)
solana-foundation/developer-content: docs/intro/overview.md

---
sidebarLabel: Overview
title: "What is Solana?"
sidebarSortOrder: 1
description:
  "Solana is the high-performance blockchain designed for mass adoption. Learn
  why Solana is the top choice for developers looking to build scalable
  blockchain applications."
altRoutes:
  - /docs/intro/history
  - /docs/intro
---

> Solana is a blockchain built for mass adoption. It's a high performance network
> that is utilized for a range of use cases, including finance, payments, and
> gaming. Solana operates as a single global state machine, and is open,
> interoperable, and decentralized.

Reviewer: ```suggestion
Solana is a high performance blockchain used for finance, payments, gaming and more. Solana operates as a single global state machine, and is open,
interoperable, and decentralized.
```

Author: This is actually from our internal boilerplate. It's how we're supposed to write about Solana 🤷

> ## Why Solana?
> 
> Designed to scale, Solana is purpose built for blockchain applications to reach
> millions of users. Instead of worrying about optimizing for the blockchain
> layer, developers can focus on building their application to reach product
> market fit. Not only can it scale now for blockchain application needs, but the
> network continues to be optimized with user experience in mind.
> 
> Building the best user experience in an application is a top priority for
> developers. With blockchains, the user experience is often limited by the
> underlying technology, causing slow response times and high fees. Solana's low
> fees and 400ms confirmation times enable developers to build applications that
> are user friendly and accessible to everyone.

Reviewer: ```suggestion
## Why Solana?

Solana's low fees and 400ms confirmation times enable developers to build applications that are responsive and scale to millions of users. Instead of worrying about optimizing for blockchains with slow response times and high fees, Solana allows developers to focus on building their application to reach product
market fit. 
```

Reviewer: I think we can say this with less words, so people can see the amazing list of features in the next section.

## Solana Features

| Feature                      | Description                                                                                                                                                                                                                   |
| ---------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | On-chain Program Development | The ability to develop and deploy programs on-chain. Users can interact with these programs permissionlessly, removing the need for middleman servers to build applications.                                                  |

Reviewer: 'onchain' - do a find and replace.

| 400ms Slot Times             | Every transaction sent by users on Solana is confirmed in a block. With a target slot time of 400ms for each block, interactions by users do not have to wait, providing the best in class user experience of any blockchain. |

> | Low Fees                     | Fees on Solana are well known to be low. With a median fee of 0.00064 SOL per transaction, Solana enables building for users across the world regardless of background.                                                       |

Reviewer: Worth adding a mention of compressed NFTs here? Or should we keep it short?

Author: Trying to avoid getting into too much jargon

| High Throughput              | Scaling to thousands of transactions per second, Solana is built to scale with your application's user needs.                                                                                                                 |

> 

Reviewer: | Large user community | Solana is the [most active blockchain for stablecoin transactions](https://visaonchainanalytics.com/transactions). |

Reviewer: One big advantage we have over other current-gen networks is the size of our user base and developer base. We should definitely mention this. If I was picking a chain, going to where the users are is huge. 

Not sure if maybe we want to add composability too?
